packages/accounts/src/msca/plugins/session-key/extension.ts

```diff
@@ -0,0 +1,73 @@
+import type {
+  Address,
+  ISmartAccountProvider,
+  SupportedTransports,
+} from "@alchemy/aa-core";
+import type { IMSCA } from "../../types.js";
+import { SessionKeyPlugin } from "./plugin.js";
+import { buildSessionKeysToRemoveStruct } from "./utils.js";
+
+export const ExtendedSessionKeyPlugin = {
```

Comment:
other methods we should include here you can think of? @moldy530

Reply:
Once we have the latest ABIs we'll probably also want to support `rotateSessionKey`